.github/actions/create-jira-issues-for-community-activities/action.yml

```diff
@@ -33,10 +33,10 @@ runs:
       shell: pwsh
       run: |
         $output = "${{ github.event.pull_request }}" -eq "" ? "False" : (Confirm-PullRequestTitle "$($Env:PR_TITLE)")
-        Set-Output "is-issue-pr" $output
+        Set-Output -Key "is-issue-pr" -Value $output
 
     - name: Login to Jira
-      if: steps.issue-related-pr.outputs.is-issue-pr == 'False'
+      if: ${{ steps.issue-related-pr.outputs.is-issue-pr }} == 'False'
```

Comment:
These changes, as well as those below, shouldn't be needed. Same in .github/actions/publish-nuget/action.yml.

Reply:
If I remember correctly, it was only working with this change, but I will check to make sure.

Reply:
Yes, I checked, and this is only working with this change.

Reply:
This would go against the logic of these expressions (either the whole expression needs to be wrapped into `${{ }}`, or rather, the `, if`s in particular, don't need this at all). So, please check this with a minimal repro: A workflow that in one step writes "False" to the output, and then checks in the next one. Try what happens if you change the output to "True".

Reply:
The root of the problem was that I was using `$GITHUB_OUTPUT` in the script but it should have been `$Env:GITHUB_OUTPUT ` :dizzy_face:

Reply:
Indeed!